applications/vessel-history/src/features/workspace/workspace.slice.ts

```diff
@@ -0,0 +1,59 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
```

Comment:
I'd review what things are used from here and try to cleanup as much as possible because there are some things only from fishing-map features

Reply:
Done, thanks!